src/lib/pci_virtio_sock.c

```diff
@@ -263,6 +264,7 @@ struct pci_vtsock_sock {
 
 	uint32_t buf_alloc;
 	uint32_t fwd_cnt;
+	uint32_t credit_limit;
```

Comment:
This is intentionally unused (apart from to set it) at this point in the series?